tests/data/reco_pixel_single/conversion_scripts/get_toclient_msg_infile.py

```diff
@@ -25,18 +27,29 @@ def main():
     args = parser.parse_args()
 
     with open(args.pframe_pkl, "rb") as f:
-        pframe = pickle.load(f)
+        depickled = pickle.load(f)
 
     # When extracting the debug .pkl from ewms-pilot, the in- pickles already contain the full message.
     # Do we need to support "bare" pframes pickles at all?
     # For the moment, this is a workaround.
```

Comment:
Does this comment still apply?

Reply:
It does not. Really, the entire script is useless now since the test data is all of the same format. Do you see any reason not to rm the file?

Reply:
update: rm'd the whole dir (https://github.com/icecube/skymap_scanner/pull/271/commits/1cfc2db27295a3b28edcc5a46d74689394439e22), but I'll restore it if needed